Internals/Errors/Messages.js

```diff
@@ -22,6 +22,19 @@ const Messages = {
 	UNAUTHORIZED_USER: user => `"${user.tag}" is not part of the sudo or normal maintainer list.`,
 	SHARED_INVALID_MODE: (mode, command) => `"${mode}" is an invalid mode for command ${command}`,
 
+	// CLI errors
+	CLI_PARAM_INVALID: (paramName, receivedType, allowedTypes) => {
+		function mapTo (string) {
```

Comment:
`const mapTo = string => `

Reply:
show tttie wuv for not throwing es5 functions in trash